tests/test-cases/Group23-VIC-Machine-Service/23-03-VCH-Create.robot

```diff
@@ -54,10 +61,36 @@ Inspect VCH Config ${name}
 Get VCH ${name}
     Get Path Under Target    vch
     ${id}=    Run    echo '${OUTPUT}' | jq -r '.vchs[] | select(.name=="${name}").id'
+    Set Test Variable    ${id}
 
     Get Path Under Target    vch/${id}
 
 
+Pull Busy Box
+    [Arguments]    ${docker_params}    ${expected_rc}    ${expected_output}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker -H ${docker_params} pull busybox
+
+    Should Be Equal As Integers    ${rc}    ${expected_rc}
+    Should Contain    ${output}    ${expected_output}
+
+
+Get Docker Host Params
+    [Arguments]    ${vch}    ${vch_name}
+    Run Keyword And Continue On Failure    Wait Until Keyword Succeeds    20x    10s    Get Docker Params API    ${vch}    ${vch_name}
+    Run Keyword If    "${docker_host}" == "null"    Run VIC Machine Inspect Command   ${vch_name}
+    ${vch_params}=    Get Environment Variable    VCH-PARAMS    ${EMPTY}
+    Run Keyword if    "${vch_params}" != "${EMPTY}"    Extract Docker IP And Port    ${vch_params}
+
+
+Extract Docker IP And Port
+    [Arguments]    ${vch_params}
+    Log To Console    Get Docker Params API did not work, use Run VIC Machine Inspect Command instead for docker_host info
+    @{hostParts}=    Split String    ${vch_params}
+    ${docker_host}=    Strip String    @{hostParts}[1]
```

Comment:
I didn't specify a separator here since the doc says `If a separator is not given, any whitespace string is a separator. In that case also possible consecutive whitespace as well as leading and trailing whitespace is ignored.` 
Is there a best practice here to indicate ${Space} to make it obvious?